errorfinder/errorfinder.go

```diff
@@ -0,0 +1,139 @@
+package errorfinder
+
+import (
+	"bufio"
+	"regexp"
+	"strings"
+)
+
+// NSError ...
+type NSError struct {
```

Comment:
Do we need to expose this struct?

Reply:
Good point. It is just for internal usage. I have removed it.